docs/_config.yml

```diff
@@ -74,7 +74,6 @@ sidebar_categories:
     - packages/random
     - packages/server-render
     - packages/spacebars
-    - packages/underscore
```

Comment:
I think for docs you will also need to replace underscore in the Meteor theme.

Reply:
Sorry but I don't understand. Replace underscore as in some config file or Meteor theme code?